Makefile

```diff
@@ -536,3 +536,11 @@ requires-license:
 ifndef LICENSE_KEY
 	$(error LICENSE_KEY is required)
 endif
+
+.PHONY: test-engine-extra-metadata
+test-engine-extra-metadata:
+	MEMSQL_SERVER_VERSION=${SERVER_VERSION_PREVIEW} \
+	RELEASE_CHANNEL=dev \
+	RELEASE_METADATA_BUCKET=helios-infra-engine-releases-metadata \
+	RELEASE_METADATA_BUCKET_REGION=us-east-1 \
```

Comment:
bucket and region should come from env variables (configured in circle ci)

Reply:
It appears that apart from exposing those two env variables via CircleCI we will need also to expose the static secrets that allows us to access the S3 bucket in CircleCI. Currently jobs are failing with the following error:
```
MEMSQL_SERVER_VERSION=8.5.1-417dd55bd4 \
RELEASE_CHANNEL=dev \
RELEASE_METADATA_BUCKET=helios-infra-engine-releases-metadata \
RELEASE_METADATA_BUCKET_REGION=us-east-1 \
./assets/release-metadata/release-metadata.py
ERROR:root:Missing required environment variable RELEASE_METADATA_AWS_ACCESS_KEY_SECRET
Traceback (most recent call last):
  File "./assets/release-metadata/release-metadata.py", line 55, in <module>
    releaseMetadataAWSAccessKeySecret = _must_get_env("RELEASE_METADATA_AWS_ACCESS_KEY_SECRET", redact_log=True)
  File "./assets/release-metadata/release-metadata.py", line 15, in _must_get_env
    raise Exception("Environment variable {} is required".format(var))
Exception: Environment variable RELEASE_METADATA_AWS_ACCESS_KEY_SECRET is required

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./assets/release-metadata/release-metadata.py", line 72, in <module>
    if os.path.exists(releaseMetadataFile):
NameError: name 'releaseMetadataFile' is not defined
make: *** [Makefile:571: test-engine-extra-metadata] Error 1

Exited with code exit status 2
```

https://app.circleci.com/pipelines/github/memsql/deployment-docker/662/workflows/8de566dc-3140-4a60-b253-6f182b5f293a/jobs/3960

Reply:
Done!